doc/refman/RefMan-ltac.tex

```diff
@@ -600,6 +613,11 @@ \section{Semantics}
 
 \ErrMsg \errindex{Cannot solve the goal}
 
+\variant {\tt solve {\tacexpr}}
+
+This is an Ltac alias that gives a primitive access to the {\tt solve} tactical.
+See the {\tt first} tactical for more information.
```

Comment:
Let's try to fix that. I can't even understand from the makefile output the error on my machine.